roles/confluent.kerberos/templates/krb5.conf.j2

```diff
@@ -0,0 +1,30 @@
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[libdefaults]
+ default_realm = {{ realm_name|upper() }}
```

Comment:
bug?? should be realm not realm_name

Reply:
created https://confluentinc.atlassian.net/browse/PLAT-601 to track

Reply:
Changes merged and corrected.